application/templates/index.html

```diff
@@ -9,6 +9,7 @@ <h2>Single-Lane Sequencing Graphs</h2>
     <h3><a href="/single-lane-ts">Single-Lane Targeted Sequencing</a></h3>
     <h3><a href="/single-lane-wgs">Single-Lane WGS</a></h3>
     <h3><a href="/single-lane-rna">Single-Lane RNA-seq</a></h3>
+	<h3><a href="/bcl2fastq-index-qc">Bcl2fastq Index QC</a></h3>
```

Comment:
I'm not sure if "Single-Lane Sequencing Graphs" is the best place for this, as it represents data at a run level. What do you think about putting it under a `Single-Run Sequencing Graphs` h2 header?